src/GeneralStateTestsFiller/stCreateTest/CreateAddressWarmAfterNonceMaxFiller.yml

```diff
@@ -0,0 +1,88 @@
+CreateAddressWarmAfterNonceMax:
+  _info:
+    comment: |
+      Invokes failing CREATE (nonce value is maximum) and checks
+      if the create address is considered warm in the follow up call as required by EIP-2929.
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 1
+    currentGasLimit: 30000000
+    currentNumber: 2
+    currentTimestamp: 999
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    # Tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      nonce: 0
+      balance: 100000001
+      storage: {}
+      code: ''
+
+    00000000000000000000000000000000000c0deC:
+      nonce: "0xffffffffffffffff"
+      balance: 0
+      storage: {}
+      code: |
+        :yul
+          object "C" {
+            code {
+              sstore(0, create(0, 0, 0))
+              // Call the create address.
+              sstore(1, call(0, 0xf4eb6941ffbbffd112b83bda0d43f1c9d98d450a, callvalue(), 0, 0, 0, 0))
+              stop()
+            }
+          }
+  
+  transaction:
+    data:
+      - ''
+    gasLimit:
+      - 10000000
+    gasPrice: 10
+    nonce: 0
+    to: "00000000000000000000000000000000000c0deC"
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    value:
+      - 0
+      - 1
+
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value:
+          - !!int 0
+      network:
+        - ">=London"
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          nonce: 1
+        00000000000000000000000000000000000c0deC:
+          nonce: "0xffffffffffffffff"    # Create failed because nonce value is maximum.
+          balance: 0
+          storage:
+            0: 0    # Create failed
+            1: 1    # Call to create address successful
+        f4eb6941ffbbffd112b83bda0d43f1c9d98d450a: # Create address
+          shouldnotexist: true
+
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value:
+          - !!int 1
+      network:
+        - ">=London"
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          nonce: 1
+        00000000000000000000000000000000000c0deC:
+          nonce: "0xffffffffffffffff"    # Create failed because nonce value is maximum.
+          storage:
+            0: 0    # Create failed
+            1: 1    # Call to create address successful
+        f4eb6941ffbbffd112b83bda0d43f1c9d98d450a: # Create address
```

Comment:
Here as well. The gas price of call we expect to change (become lower) since its warmed up